### PR TITLE
Screen reader unit tests for PhotoField

### PR DESCRIPTION
**Pull request by jbalboni**
I attempted to write these with DefinitionTester, but never really got it to work correctly. So they're written as shallow rendered component tests to get around some of the mocking we'd have to do otherwise.

### CODE REVIEW

**Review by rianfowler — APPROVED** [review]

**Comment by rianfowler** [comment]
Do we want to refactor `PhotoField` into smaller components before moving to VRE?

**Comment by cehsu** [comment]
Thanks for the reminder @rianfowler, issue created [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/8805).

File: src/js/common/utils/helpers.js [hunk 1]
```diff
@@ -153,10 +153,12 @@ export function scrollToFirstError() {
 }
 
 export function scrollAndFocus(errorEl) {
-  const currentPosition = window.pageYOffset || document.documentElement.scrollTop || document.body.scrollTop || 0;
-  const position = errorEl.getBoundingClientRect().top + currentPosition;
-  Scroll.animateScroll.scrollTo(position - 10, getScrollOptions());
-  focusElement(errorEl);
+  if (errorEl) {
```
Comment (cehsu) [inline src/js/common/utils/helpers.js]: Should we be using refs to obtain this instead? If so, we could add this to the refactor ticket.

**Review by cehsu — APPROVED** [review]

**Comment by jbalboni** [comment]
@rianfowler Let's do the VRE form we have in progress before the PhotoField refactoring.

File: src/js/common/utils/helpers.js (hunk 1 above)
Comment (jbalboni) [reply src/js/common/utils/helpers.js]: Yeah, we should probably use refs for the error message in PhotoField. I'll add it to the refactor ticket.

**Pull request merged**